src/main/fc/rc.c

```diff
@@ -85,12 +85,14 @@ enum {
 #ifdef USE_FEEDFORWARD
 static float feedforwardSmoothed[3];
 static float feedforwardRaw[3];
+static uint16_t feedforwardAveraging; 
 typedef struct laggedMovingAverageCombined_s {
     laggedMovingAverage_t filter;
     float buf[4];
 } laggedMovingAverageCombined_t;
 laggedMovingAverageCombined_t  feedforwardDeltaAvg[XYZ_AXIS_COUNT];
 
+static pt1Filter_t feedforwardYawHoldLpf;
```

Comment:
line break
```suggestion
static pt1Filter_t feedforwardYawHoldLpf;

```

Reply:
OK

Reply:
@KarateBrot I think I added the line break properly, but this still shows as a review request.  If OK, maybe this is ready to be approved?